nixos/modules/services/web-apps/alps.nix

```diff
@@ -122,8 +122,10 @@ in {
         RestrictNamespaces = true;
         RestrictRealtime = true;
         RestrictSUIDSGID = true;
+        SocketBindAllow = cfg.port;
+        SocketBindDeny = "any";
         SystemCallArchitectures = "native";
-        SystemCallFilter = [ "@system-service @resources" "~@privileged @obsolete" ];
+        SystemCallFilter = [ "@system-service" "~@privileged @obsolete" ];
```

Comment:
It's prefixed by `~` to remove all `@privileged` and `@obsolete` syscalls from the allowed set of `@system-service`.

Reply:
Didn't realize that it is interpreted in this way, what about writing it as `"~@privileged" "~@obsolete"` which is more readable.

Reply:
I grepped through `nixpkgs` for `@privileged` and I can find both `"~@privileged @obsolete"` and `"~@privileged" "~@obsolete"` with the former one maybe being slightly more common but not by much. However, since it's not easily possible to tell what is going on without reading the systemd documentation anyway, I think both are equally unreadable.